index.js

```diff
@@ -153,39 +210,87 @@ function updateDescription(newDescription) {
   description.textContent = newDescription;
 }
 
-function changeInterfaceColors(buttonColor, shadowColor, ellipseColor, hoverColor) {
-  ellipse.style.setProperty('--bg-color', ellipseColor);
-  prevBtn.style.setProperty('--bg-color', buttonColor);
-  prevBtn.style.setProperty('--shadow-bg', shadowColor);
-  prevBtn.style.setProperty('--hover-color', hoverColor);
-  nextBtn.style.setProperty('--bg-color', buttonColor);
-  nextBtn.style.setProperty('--shadow-bg', shadowColor);
-  nextBtn.style.setProperty('--hover-color', hoverColor);
-  button.style.setProperty('--bg-orange', buttonColor);
-  button.style.setProperty('--shadow-bg', shadowColor);
-  button.style.setProperty('--hover-color', hoverColor);
-
-  items.forEach(item => {
-    item.style.setProperty('--hover-color', hoverColor);
-  });
+class submenu extends HTMLElement {
```

Comment:
Por lo normal el nombre de una clase se escribe con la primera letra en mayuscula
```suggestion
class Submenu extends HTMLElement {
```